### DISCUSSION TITLE
Offline mode (testing)

### DISCUSSION THREAD

**Discussion by donl** [appy-one/acebase-client#22] [Q&A] [answered]
Is there a way to switch the client into an offline mode?

I'd like to be able to test functionality switching from offline to online, etc. while not having to manually disconnect hardware.

TIA!

**Comment by appy-one** [comment, marked as answer]
Yes, you can by calling `client.disconnect()` and `client.connect()`. If you want to disable automatically connecting upon startup, add `autoConnect: false` to your client config
